app/templates/auth/request-password-reset.html

```diff
@@ -33,15 +33,23 @@ <h1 class="govuk-heading-xl">Reset password</h1>
       <div class="govuk-grid-column-two-thirds">
         {{ form.hidden_tag() }}
 
-        {% with question = form.email_address.label,
-                name = "email_address",
-                error = errors.get('email_address', {}).get('message'),
-                hint = form.email_address.hint,
-                value = form.email_address.data,
-                autocomplete = "off"
-        %}
-          {% include "toolkit/forms/textbox.html" %}
-        {% endwith %}
+        {{ govukInput({
+          "label": {
+            "text": form.email_address.label.text,
+          },
+          "hint": {
+            "text": form.email_address.hint,
+          },
+          "errorMessage": errors.get('email_address', {}).get('errorMessage'),
+          "id": "input-email_address",
+          "type": "email",
+          "name": "email_address",
+          "value": form.email_address.data,
+          "attributes": {
+            "autocomplete": "off",
+            "spellcheck": "false",
```

Comment:
Do we only need `spellcheck: false` for email fields? What does this gain us?

Reply:
This is based on advice from GOV.UK Design System here https://design-system.service.gov.uk/patterns/email-addresses/#help-users-to-enter-a-valid-email-address:

> setting the spellcheck attribute to false so that browsers do not spellcheck the email address

I don't know how big of a difference it makes, as according to [the MDN article on spellcheck](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/spellcheck) it's not clear whether most mainstream browsers spellcheck input fields. I can't see including it doing any harm however.

If you want I'm happy to take it out.

Reply:
If that's the guidance then I am happy 👍